README.md

```diff
@@ -16,15 +16,15 @@ If you see a step below that could be improved (or is outdated), please update t
 
 - We recommend using [nvm](https://github.com/creationix/nvm): `nvm install && nvm use`.
 
-2. Make sure you have a PostgreSQL database available
+2. Make sure you have a PostgreSQL database available.
 
-- Check the version: 11.0, 10.3, 9.6.8, 9.5.12, 9.4.17, 9.3.22 or newer
+- Check the version (in production we're currently running 13.3)
 - Check that the [PostGIS](https://postgis.net/install/) extension is available
 - More info in our [PostgreSQL Database](docs/postgres.md) documentation
 
-3. For [node-gyp](https://github.com/nodejs/node-gyp), make sure you have Python 2 available and configured as the active version.
+3. For [node-gyp](https://github.com/nodejs/node-gyp), make sure you have Python 3 available and configured as the active version.
 
-- You can use [pyenv](https://github.com/pyenv/pyenv) to manage Python versions.
+- You can use [conda](https://www.anaconda.com/) or [pyenv](https://github.com/pyenv/pyenv) to manage Python versions.
```

Comment:
Never used so far but sounds good!

Reply:
Conda is my go-to package manager because while it's built for Python, it's flexible with lots of other languages and therefore more robust than `pyenv`. In my experience a lot of Python coders use Conda to manage environments, so I spent some time getting it to work with `node-gyp` and PostreSQL to set up end-to-end tests for another pull request. It turned out to be pretty straightforward 😊